apk-generator/v2/app/utils/notification.py

```diff
@@ -21,16 +22,47 @@ def send(to, subject, message, file_attachment, via_api):
         :param via_api:
         :return:
         """
-        pass
+
+        email_service = current_app.config['EMAIL_SERVICE']
+
+        payload = {
+            'to': to,
+            'from': current_app.config['SMTP_USERNAME'],
```

Comment:
The smtp username may not always be an email address. And why would there be an smtp username if the the user wants to use sendgrid. ? 

So, have a separate variable called as `FROM_EMAIL`